app/controllers/settings/imports_controller.rb

```diff
@@ -11,24 +11,27 @@ def show
   end
 
   def create
-    @import = Import.new(import_params)
+    if params[:import].nil?
```

Comment:
I don't know why it should be done in `before_action`. It is a role of `create` to `render :show`.